yarn.lock

```diff
@@ -7723,6 +7723,11 @@ onetime@^2.0.0:
   dependencies:
     mimic-fn "^1.0.0"
 
+opencc-js@^1.0.4:
+  version "1.0.4"
+  resolved "https://registry.yarnpkg.com/opencc-js/-/opencc-js-1.0.4.tgz#aab415a181e09512418d513244a0d27bef3ba8df"
+  integrity sha512-0vPnzCrUhmEKDakju7prFD851PW4LPyOMBUCx4g9ndu6s0X4s9bULJRe/Ns+kZVh9i7qPhC53xm4HeON1hHpVg==
+
```

Comment:
Is it possible to avoid adding this engine as a dependency?

Reply:
You means don't add `opencc-js`?

Reply:
yes, don't add it inside i18n Ally, but have it available locally via some URL for example.

Reply:
But this feature need this， opencc-js is a local translator. This is not a remote service.